policyengine_uk_data/datasets/frs/frs.py

```diff
@@ -151,6 +151,24 @@ def add_random_variables(self, frs: dict):
             value = simulation.calculate(variable, self.time_period).values
             frs[variable] = {period: value for period in INPUT_PERIODS}
 
+        extended_would_claim = (
+            np.random.random(len(simulation.calculate("benunit_id"))) < 0.53
+        )
+        tfc_would_claim = (
+            np.random.random(len(simulation.calculate("benunit_id"))) < 0.39
+        )
+        universal_would_claim = (
+            np.random.random(len(simulation.calculate("benunit_id"))) < 0.32
+        )
+        targeted_would_claim = (
+            np.random.random(len(simulation.calculate("benunit_id"))) < 0.59
+        )
+        for period in INPUT_PERIODS:
+            frs["would_claim_extended_childcare"] = extended_would_claim
```

Comment:
You want to assign this for each of the time periods: all PE datasets have the form:
{ employment_income: {2025: [100k, 30k, ...], 2026_if_provided:...}